cmd/sovereignnode/chainSimulator/tests/bridge/bridge_test.go

```diff
@@ -55,12 +75,27 @@ func TestSovereignChainSimulator_DeployBridgeContractsThenIssueAndDeposit(t *tes
 
 	nodeHandler := cs.GetNodeHandler(core.SovereignChainShardId)
 
-	wallet, err := cs.GenerateAndMintWalletAddress(core.SovereignChainShardId, chainSim.InitialAmount)
+	initialAddress := "erd1l6xt0rqlyzw56a3k8xwwshq2dcjwy3q9cppucvqsmdyw8r98dz3sae0kxl"
+	initialAddrBytes, err := cs.GetNodeHandler(0).GetCoreComponents().AddressPubKeyConverter().Decode(initialAddress)
+	require.Nil(t, err)
+
+	err = cs.SetStateMultiple([]*dtos.AddressState{
+		{
+			Address: initialAddress,
+			Balance: "10000000000000000000000",
+		},
+	})
+	require.Nil(t, err)
+
+	wallet := dtos.WalletAddress{Bech32: initialAddress, Bytes: initialAddrBytes}
+
+	expectedESDTSafeAddressBytes, err := nodeHandler.GetCoreComponents().AddressPubKeyConverter().Decode(outGoingSubscribedAddress)
 	require.Nil(t, err)
-	nonce := uint64(0)
 
-	bridgeData := DeploySovereignBridgeSetup(t, cs, esdtSafeWasmPath, feeMarketWasmPath)
+	bridgeData := DeploySovereignBridgeSetup(t, cs, wallet, esdtSafeWasmPath, feeMarketWasmPath)
+	require.Equal(t, expectedESDTSafeAddressBytes, bridgeData.ESDTSafeAddress)
 
+	nonce := uint64(5)
```

Comment:
This is not really ok because the number of deploy transactions will be changed in the future. Just create another wallet and start with nonce 0 (do all the operations as a new user, we don't need to do them from bridge owner wallet)

Reply:
I was aware of this limitation, but didn't want to do any more refactor already in this test.
Made a common func to return the acc nonce